src/node.rs

```diff
@@ -2165,11 +2165,11 @@ impl<S: Send + Sync + 'static + ForkSource + std::fmt::Debug> EthNamespaceT for
     ///
     /// If the from `fromBlock` or `toBlock` option are equal to "latest" the filter continually appends logs for newly mined blocks.
     /// Topics are order-dependent. A transaction with a log with topics [A, B] will be matched by the following topic filters:
-    ///     * [] "anything"
-    ///     * [A] "A in first position (and anything after)"
-    ///     * [null, B] "anything in first position AND B in second position (and anything after)"
-    ///     * [A, B] "A in first position AND B in second position (and anything after)"
-    ///     * [[A, B], [A, B]] "(A OR B) in first position AND (A OR B) in second position (and anything after)"
+    ///     * \[\] "anything"
```

Comment:
Oh fascinating... never thought about needing to escape square brackets for comments O_o

Reply:
Yeah else they are picked up by rustdocs and it attempts to resolve the tokens (and not finding them)